components/kyma-environment-broker/Makefile

```diff
@@ -50,3 +50,19 @@ testing-with-database-network:
 
 clean-up:
 	@docker network rm $(TESTING_DB_NETWORK) || true
+
+run-local:
```

Comment:
a full blown shell script wrapped in a makefile? Maybe we should extract it into a .sh file and execute it in the makefile